packages/api/src/services/labels.ts

```diff
@@ -41,21 +41,18 @@ export const findOrCreateLabels = async (
   labels: CreateLabelInput[],
   userId: string
 ): Promise<Label[]> => {
-  // create labels if not exist
-  await authTrx(
-    async (tx) =>
-      tx.withRepository(labelRepository).createLabels(labels, userId),
-    undefined,
-    userId
-  )
-
-  // find labels
   return authTrx(
-    async (tx) =>
-      tx.withRepository(labelRepository).findByNames(
-        labels.map((l) => l.name),
-        userId
-      ),
+    async (tx) => {
+      const repo = tx.withRepository(labelRepository)
+      // create labels if not exist
+      await repo.createLabels(labels, userId)
+
+      // find labels by names
+      return repo.findBy({
```

Comment:
is this part case insensitive?

Reply:
No, this part is case sensitive so we ensure only one RSS label would be matched if there is another `rss` label created by users